pdf1/intermidiate_functions.py

```diff
@@ -0,0 +1,38 @@
+import random
+
+
+def addition(first_number=0, second_number=0) -> int:
+    return first_number + second_number
+
+
+def hii_name(name: str) -> str:
+    print(f"Hello {name}! Great to see u")
+
+
+def quadratic_equation(a_number: int, b_number: int, c_number: int) -> int:
+    root = (b_number**2) - (4 * a_number * c_number)
+
+    if root > 0:
+        first_result = (-b_number + (root**0.5)) / (2 * a_number)
+        second_result = (-b_number - (root**0.5)) / (2 * a_number)
+        return first_result, second_result
+    else:
+        print("roots are imaginary")
+
+
+def random_number(first_number: float or int, second_number: float or int) -> int:
+    if type(first_number) and type(second_number) == float:
+        return random.randint(first_number, second_number)
```

Comment:
good answer, it is a correct answer, but you should also read about how to random a double.
As a bonus you could read about how to specify multiple return types